docs/maui/native-library-interop/get-started.md

```diff
@@ -92,9 +92,16 @@ On the native side, make updates in _template/macios/native/NewBinding/NewBindin
 
 Back on the .NET side, we are now ready to interop with the native library:
 
-1. Run `dotnet build` from _template/macios/NewBinding.MaciOS.Binding_ to test everything is plugged in correctly and good to go. If successful, you should see the generated C# bindings at _template/macios/native/NewBinding/bin/Release/net8.0-ios/sharpie/NewBinding/ApiDefinitions.cs_.
-1. Update the contents of _template/macios/NewBinding.MaciOS.Binding/ApiDefinition.cs_ by replacing it with the contents of _template/macios/native/NewBinding/bin/Release/net8.0-ios/sharpie/NewBinding/ApiDefinitions.cs_.
-1. Run `dotnet build` from _template/macios/NewBinding.MaciOS.Binding_ again.
+1. Run `dotnet build` from _template/macios/NewBinding.MaciOS.Binding_ to test everything is plugged in correctly and good to go.
+1. Optionally use objective sharpie to generate an updated ApiDefinitions.cs file that reflects any changes in your swift code.
+    1. Run `sharpie xcode -sdks` to get a list of valid target SDK values for the bind command. Select the value that aligns with the platform and version you are targeting to use with the next command, for example `iphoneos18.0`.
+    1. Run `sharpie bind` against the header files in the xcframework created by the binding project:
+        ```
+        cd _template/macios/native/NewBinding/bin/Debug/net9.0-ios/NewBinding.MaciOS.Binding.resources/NewBindingiOS.xcframework/ios-arm64/NewBinding.framework &&
```

Comment:
```suggestion
```

Reply:
Remove the cd part of the command, and suggest navigating to location as a separate path beforehand

Reply:
Updated